src/web/core/src/main/java/org/geoserver/web/data/store/WMSStoreNewPage.java

```diff
@@ -8,14 +8,15 @@
 import java.io.IOException;
```

Comment:
Yeah, this file still states only 2014. Anyways, as said, no big deal, it's quite likely that we are going to have to modify these files further during the sprint.